packages/react-shopify-app-route-propagator/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+<!-- Unreleased changes should go to UNRELEASED.md -->
+
+---
+
+## 2.0.0 - 2019-01-09
```

Comment:
The last release should had been version 2 but it was missed. Can we make this 2.0 please?